app/views/home/index.html.erb

```diff
@@ -6,12 +6,40 @@
   </p>
 </div>
 
-<h2><%= t('latest_projects') %></h2>
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-md-4">
+      <h2><%= t('latest_projects') %></h2>
 
-<ul>
-  <% cache do %>
-    <% @latest_projects.each do |project| %>
-      <li><%= link_to(project.name, project_path(project.id)) %></li>
-    <% end %>
-  <% end %>
-</ul>
+      <ul>
+        <% cache do %>
```

Comment:
Again:
> It already gets expired and works and is consistent with the current code, thus not a merge blocker if your concern is correctness.

I can assure you this was tested for file persistence caching and memcached used both by development/testing and production. Or do you have a counter example?

If you are curious about other engines, please create question issues properly.